### ISSUE TITLE
data from embedded infoboxes no longer included

### ISSUE THREAD

**Issue by tmtmtmtm** [spencermountain/wtf_wikipedia#506]
When an infobox embeds a second infobox, `wtf_wikipedia` used to handle this. In the latest release it doesn't seem to.

So, for example, running `wtf_wikipedia "Dee-Ann Kentish-Rogers" | jq -r .` with both 9.1.0 and 10.0.5 and diffing the output, I get:

```diff
104a105,115
>         },
>         {
>           "template": "infobox",
>           "type": "officeholder",
>           "data": {
>             "embed": {},
>             "office1": {},
>             "term_start1": {},
>             "predecessor1": {},
>             "party": {}
>           }
109,149d119
<           "embed": {
<             "text": "yes"
<           },
<           "office1": {
<             "text": "Member of the House of Assembly for Valley South",
<             "links": [
<               {
<                 "text": "House of Assembly",
<                 "type": "internal",
<                 "page": "Anguilla House of Assembly"
<               },
<               {
<                 "text": "Valley South",
<                 "type": "internal",
<                 "page": "Valley South (Anguilla House of Assembly Constituency)"
<               }
<             ]
<           },
<           "term_start1": {
<             "text": "30 June 2020"
<           },
<           "predecessor1": {
<             "text": "Victor Banks",
<             "links": [
<               {
<                 "type": "internal",
<                 "page": "Victor Banks"
<               }
<             ]
<           },
<           "party": {
<             "text": "Anguilla Progressive Movement",
<             "links": [
<               {
<                 "type": "internal",
<                 "page": "Anguilla Progressive Movement"
<               }
<             ]
<           }
<         },
<         {
```

(I'm happy to bsearch through intermediate versions to see when this changed, if needed)

**Comment by tmtmtmtm** [comment]
Looks like this changed in 10.0.3

**Comment by spencermountain** [comment]
hey Tony, thanks. I've been thinking abt this - I'm not sure what's best to do.
We've always had a vague distinction that Infoboxes are seperate templates, and one aspect of them is they are not nested. 
It's sort of intentional, but I dunno. maybe it's not helpful. I think we're seeing more and more nested infoboxes by wikipedia editors, perhaps.

A fix would be easy, on [this line](https://github.com/spencermountain/wtf_wikipedia/blob/dev/src/template/sortOut.js#L30).
If anyone has any opinions on this, please weigh in!


for reference:
```js
str = `{{Infobox pageant titleholder
  | name         = Dee-Ann Kentish-Rogers
  | birth_date   = {{birth date and age|1993|01|13|df=yes}}
  | caption      = Kentish-Rogers in a 2020 political advertisement
  | measurements = 
  | module={{Infobox officeholder
  |name         = 
  |term_start1  = 30 June 2020
  |predecessor1  = [[Victor Banks]]
  |successor1    =
  }}
  }}
`
let doc = wtf(str)
console.log(doc.infoboxes())
```
cheers

**Comment by tmtmtmtm** [comment]
@spencermountain Thanks for looking into this. I think nested infoboxes are quite common, at least in the areas I look most at — primarily politicians.

I've seen at least two main cases where this happens. The first, like Dee-Ann Kentish-Rogers above, is where someone has also had a prominent career outside of politics, e.g. in sports, entertainment, the military or the like, where nested infoboxes are used to partition the data about each. (This appears to happen with royalty too, though I don't deal with those very much, so have much less of a feel for how common it is there.)

The other very common case is for senior politicians, where their earlier, more-junior political career, is hidden behind an "Expand" toggle. See, for example 
* https://en.wikipedia.org/wiki/Olaf_Scholz 
* https://en.wikipedia.org/wiki/Rishi_Sunak 
* https://en.wikipedia.org/wiki/Ursula_von_der_Leyen
* https://en.wikipedia.org/wiki/Anthony_Albanese
* https://en.wikipedia.org/wiki/Kassym-Jomart_Tokayev
* https://en.wikipedia.org/wiki/Fumio_Kishida

I just ran a test against the list of current UK Cabinet Ministers, and a substantial proportion of them (Rishi Sunak, Jeremy Hunt, James Cleverly, Stephen Barclay, Thérèse Coffey, Dominic Raab, Penny Mordaunt, Nadhim Zahawi, Michael Gove, Kemi Badenoch and Johnny Mercer), all have embedded infoboxes that the latest version chokes on.

**Comment by spencermountain** [comment]
perfect. thank you. I'll make the change today

**Comment by spencermountain** [comment]
released in 10.1.0
cheers!

**Comment by tmtmtmtm** [comment]
@spencermountain Thanks! Everything's looking good for all the cases I've tested so far 🚀
